scarb/Cargo.toml

```diff
@@ -19,6 +19,7 @@ async-trait.workspace = true
 cairo-lang-compiler.workspace = true
 cairo-lang-defs.workspace = true
 cairo-lang-diagnostics.workspace = true
+cairo-lang-executable = "2.9.1"
```

Comment:
Please add this dep to https://github.com/software-mansion/scarb/blob/main/Cargo.toml and use `cairo-lang-executable.workspace = true`. This way we make sure, we use the same version of this dep throughout all crates in the workspace.